scene/gui/graph_edit.cpp

```diff
@@ -435,8 +436,8 @@ void GraphEdit::_top_layer_input(const Ref<InputEvent> &p_ev) {
 			for (int j = 0; j < gn->get_connection_input_count(); j++) {
 
 				Vector2 pos = gn->get_connection_input_position(j) + gn->get_position();
-
-				if (pos.distance_to(mpos) < grab_r) {
+				Rect2 rect = Rect2(pos.x - grab_r * 3, pos.y - grab_r, grab_r * 6, grab_r * 2);
```

Comment:
This is pretty hacky, I wonder if there can be a better way to do this..

Reply:
I extracted the common code into create_hot_zones()

Reply:
Moved constants into default theme.